_authors/eric.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Eric Carmichael
```

Comment:
You filled in Allen's author information on Eric.md. If we switch that up, everything else looks good!

Reply:
Whoops! I'll fix that and merge it over. Thanks for catching that